ui/de-lib/src/main/java/org/iplantc/de/admin/desktop/client/metadata/view/TemplatesListingViewImpl.java

```diff
@@ -123,6 +150,7 @@ public String getKey(MetadataTemplateInfo item) {
                                                                                     appearance.deleted());
         List<ColumnConfig<MetadataTemplateInfo, ?>> columns = new ArrayList<ColumnConfig<MetadataTemplateInfo, ?>>();
         columns.add(nameCol);
+        columns.add(descriptionCol);
```

Comment:
Descriptions can be lengthy. I am not sure if displaying that as a column is good a idea.